src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -115,14 +154,26 @@ def load_arguments(self, _):
             c.argument('item_name', item_name_type)
 
     with self.argument_context('backup protection backup-now') as c:
-        c.argument('retain_until', type=datetime_type, help='The date until which this backed up copy will be available for retrieval, in UTC (d-m-Y).')
+        c.argument('retain_until', type=datetime_type, help='The date until which this backed up copy will be available for retrieval, in UTC (d-m-Y).', options_list=['--retain-until', '-rt'])
```

Comment:
is -rt option from PM spec?

Reply:
No added by me. Will ask PM to confirm all shorthands once